java/i18n/src/org/netbeans/modules/i18n/wizard/AdditionalWizardPanel.java

```diff
@@ -218,7 +218,7 @@ public static class Panel extends I18nWizardDescriptor.Panel {
         private transient AdditionalWizardPanel additionalPanel;
 
         Panel () {
-            emptyLabel = new JLabel(Util.getString("TXT_HasNoAdditonal"));//NOI18N
+            emptyLabel = new JLabel(Util.getString("TXT_HasNoAdditional"));//NOI18N
```

Comment:
Is this changed in the corresponding Bundle file?

Reply:
yes